modin/pandas/test/test_io.py

```diff
@@ -16,13 +16,21 @@
 import pandas
 from pandas.errors import ParserWarning
 from collections import OrderedDict
-from modin.config import TestDatasetSize
+from modin.config import TestDatasetSize, Backend
 from modin.utils import to_pandas
 from modin.pandas.utils import from_arrow
 import pyarrow as pa
 import os
 import shutil
-import sqlalchemy as sa
+
+try:
+    import sqlalchemy as sa
+except ImportError:
```

Comment:
we should either add `sqlalchemy` in an OmniSci test environment file in this PR or create a tracker to do it in a separate PR and link it here

Reply:
agreed

Reply:
Not sure we should do these since `sqlalchemy` doesn't used by `Omnisci` backend. Do we need to add new package for tests compatibility only?

Reply:
I don't see anything wrong with adding test-only dependencies... for example, `pytest` isn't used by OmniSci, either 😉

Reply:
Ok, `sqlalchemy` was moved to the dependencies list.